### PR TITLE
[Bugfix] Resolved presentation bugs

### PR DESCRIPTION
**Pull request by palatter**
## Purpose

* Resolved missing system message icons
* Resolved message input box width
* Resolved message formatting and layout
* Resolved missing avatars
* Resolved message font colours

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/Style/IconProvider.swift [hunk 1]
```diff
@@ -11,6 +11,8 @@ enum CompositeIcon: String {
     case leftArrow = "ic_ios_arrow_left_24"
     case downArrow = "ic_fluent_arrow_down_24_filled"
     case send = "ic_fluent_send_24_filled"
+    case systemJoin = "ic_fluent_person_add"
+    case systemLeave = "ic_fluent_person_remove"
```
Comment (Leah-Xia-Microsoft) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/Style/IconProvider.swift:15]: nitpick: Do we want to be consistent about icon names either all have filled/regular suffix or leave them out?

Comment (palatter) [reply AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/Style/IconProvider.swift:15]: Good catch, I'll update the names

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift [hunk 2]
```diff
@@ -10,21 +10,20 @@ struct BottomBarView: View {
     private enum Constants {
         static let minimumHeight: CGFloat = 50
         static let focusDelay: CGFloat = 1.0
-        static let padding: CGFloat = 25.0
+        static let padding: CGFloat = 12
```
Comment (jpeng-ms) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift:13]: we might want to update typing indicator's padding as well later to make it consistent

**Review by jpeng-ms — DISMISSED** [review]
looks good!

**Review by Leah-Xia-Microsoft — DISMISSED** [review]
LGTM

**Review by Leah-Xia-Microsoft — APPROVED** [review]

**Review by jpeng-ms — APPROVED** [review]

**Review by kevinyulu — APPROVED** [review]
lgtm

**Pull request merged**